api/api/api_exception.py

```diff
@@ -18,10 +18,16 @@ def __init__(self, code: int, details: str = None):
         """
         self.code = code
         self.details = details
+        # show relative paths in exceptions
         self.exceptions = {
-            2000: f'Some parameters are not expected in the configuration file ({CONFIG_FILE_PATH})',
+            2000: 'Some parameters are not expected in the configuration file '
+                  f'(WAZUH_PATH/{to_relative_path(CONFIG_FILE_PATH)})',
             2001: 'Error creating or reading secrets file. Please, ensure '
-                  f'there is enough disk space and permission to write in {SECURITY_PATH}'
+                  'there is enough disk space and permission to write in '
+                  f'WAZUH_PATH/{to_relative_path(SECURITY_PATH)}',
+            2002: 'Error migrating configuration from old API version. '

```

Comment:
What's the error exactly? You explain the when, but not the cause of the error, so the user won't know how to proceed.

Reply:
Done!